docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.4'
```

Comment:
Again, a docker-compose.yml file is not really required here. 

It's a single service and isn't really offering you anything.

It's also forcing you to have google-app credentials in `/tmp/gcs_credentials_dir`. 

This is not required - users of this app should be free to supply creds however they want (for example they can just set GOOGLE_APPLICATION_CREDNETIALS external to this.